parts/linux/cloud-init/artifacts/components.json

```diff
@@ -367,8 +367,7 @@
       "targetContainerRuntime": "containerd"
     }
   ],
-  
-  "PackageList": [
+  "Binaries": [
```

Comment:
@Devinwong  it was a suggestion! if the team comes up with something that fits better how we call/refer to those binaries we install that are not Os Packages and container images, we should use that. 


Also, do we think that os packages/ binaries/container images will have different schema hence why we are creating different Elements ?  instead of using `componentType: binary|containerimage|package`